src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -915,7 +919,7 @@ export default {
 
 /* counter and actions */
 .app-navigation-entry__utils {
-	display: flex;
+   	display: none;
```

Comment:
This hides not only the actions but the `#counter` slot as well:
Before | After
---|---
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/5fd7811d-8b0b-44f0-9039-032d65336491) | ![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/fde58c90-259f-4009-b4b4-519a99e87cf4)

Reply:
Also, there is a linter error: `ESLint: Mixed spaces and tabs`. We use tabs for tabulation.